actors/miner/tests/prove_replica_test.rs

```diff
@@ -109,25 +109,44 @@ fn update_batch() {
         notifications
     );
 
-    // Sector 0: Even though there's no "deal", the data weight is set.
-    verify_weights(&rt, &h, snos[0], piece_size, 0);
-    // Sector 1: With an allocation, the verified weight is set instead.
-    verify_weights(&rt, &h, snos[1], 0, piece_size);
-    // Sector 2: Deal weight is set.
-    verify_weights(&rt, &h, snos[2], piece_size, 0);
-    // Sector 3: Deal doesn't make a difference to verified weight only set.
-    verify_weights(&rt, &h, snos[3], 0, piece_size);
-
-    // Check that the fees haven't changed.
-    let sectors_after = snos.iter().map(|sno| h.get_sector(&rt, *sno));
-    for (before, after) in std::iter::zip(sectors, sectors_after) {
+    let sectors_after = snos.iter().map(|sno| h.get_sector(&rt, *sno)).collect::<Vec<_>>();
+    for i in 0..sectors.len() {
```

Comment:
I kind of prefer the zip, but it's just test code so I guess performance doesn't really matter.